build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -4,12 +4,26 @@
   <PropertyGroup>
     <_SourceTopDir>..\..</_SourceTopDir>
     <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MandroidDir>$(OutputPath)\lib\mandroid</_MandroidDir>
   </PropertyGroup>
+  <ItemGroup>
+    <MonoDocItem Include="monodoc.dll" />
+    <MonoDocItem Include="monodoc.dll.mdb" />
+    <MonoDocItem Include="monodoc.dll.config" />
```

Comment:
Shouldn't `mdoc.exe` be listed in this item group as well, instead of listing it separately? It's in the same directory...